dubbo-common/src/main/java/org/apache/dubbo/rpc/model/FrameworkModel.java

```diff
@@ -123,7 +123,6 @@ protected void onDestroy() {
         // notify destroy and clean framework resources
         // see org.apache.dubbo.config.deploy.FrameworkModelCleaner
         notifyDestroy();
-        checkApplicationDestroy();
```

Comment:
why remove this?

Reply:
checkApplicationDestroy has been called on line 121
![image](https://user-images.githubusercontent.com/43363120/180680575-43055339-0169-4ea6-a700-34a2c312ca77.png)